### PR TITLE
{bio}[foss+fosscuda/2019a] CSBDeep v0.4.1 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/648beab9c8dc0dbacf2efb30e9256b4f for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/84b8a8ef8b0c5df5ac7b641fd28f0abb for a full test report.

File: easybuild/easyconfigs/c/CSBDeep/CSBDeep-0.4.1-fosscuda-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ('tifffile', '2019.7.26.2', {
+        'checksums': ['2abb91c3a23a61593c5635ac1a19f67e732b46291c305fcee0eeaad41181a13f'],
+    }),
+    (name, version, {
+        'modulename': '%(namelower)s',
+        'source_tmpl': '%(namelower)s-%(version)s.tar.gz',
+        'checksums': ['ad2cf81389b8c15eb2f38a524dc011a5331dfb91e7d3053fed0c6c808a7bdf89'],
+    }),
+]
+
```
Comment (smoors) [inline easybuild/easyconfigs/c/CSBDeep/CSBDeep-0.4.1-fosscuda-2019a-Python-3.7.2.eb]: @branfosj can you add the `sanity_check_commands = ['care_predict']` that was added to the foss version here too?

Comment (branfosj) [reply easybuild/easyconfigs/c/CSBDeep/CSBDeep-0.4.1-fosscuda-2019a-Python-3.7.2.eb]: done

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/41f0c71377d11159ff7b0e92c11f638f for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node374.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/0755820b23d290c1421551f68c0c5e0c for a full test report.

**Comment by smoors** [comment]
Going in, thanks @branfosj!

**Pull request merged**